beacon-chain/rpc/prysm/v1alpha1/node/server.go

```diff
@@ -45,6 +49,29 @@ type Server struct {
 	BeaconMonitoringPort int
 }
 
+// GetHealth checks the health of the node
+func (ns *Server) GetHealth(ctx context.Context, request *ethpb.HealthRequest) (*empty.Empty, error) {
+	_, span := trace.StartSpan(ctx, "node.GetHealth")
```

Comment:
```suggestion
	ctx, span := trace.StartSpan(ctx, "node.GetHealth")
```
If you don't overwrite the context, then the span details are not carried forward to other requests.

Reply:
Also, should you set some deadline, perhaps the server should respond within 10 seconds or give up waiting?

Reply:
added a deadline in the function if that's what you're suggesting, i don't think other similar functions have deadlines though, it might default to something for grpc. more tests were added though.